app/models/agents/http_status_agent.rb

```diff
@@ -66,7 +66,8 @@ def check
     def receive(incoming_events)
       incoming_events.each do |event|
         interpolate_with(event) do
-          check_this_url interpolated[:url], header_array(interpolated[:headers_to_save])
+          check_this_url event.payload[:url] || interpolated[:url],
+                         header_array(event.payload[:headers_to_save] || interpolated[:headers_to_save])
```

Comment:
I see that your intension is to implement the original specs, but I don't understand what the benefit is to support `event.payload[:url]`. Nobody could have been using it and liquid is always more flexible. Why should we potentially break existing use cases to support an intension that never worked?